internal/compose/compose.go

```diff
@@ -247,6 +250,63 @@ func (p *Project) Logs(opts CommandOptions) ([]byte, error) {
 	return b.Bytes(), nil
 }
 
+// WaitForHealthy method waits until all containers are healthy.
+func (p *Project) WaitForHealthy(opts CommandOptions) error {
+	// Read container IDs
+	args := p.baseArgs()
+	args = append(args, "ps")
+	args = append(args, "-q")
```

Comment:
One problem of `docker-compose ps` is that it doesn't show services that are not even started. So this method may end up succesfully when there is no container running.

This may be ok though if we only want to wait for the started services and not for others.

Reply:
That's why I have doubts once I opened this PR. Maybe we should agree on using `tianon/true` as standard? It seems to be more trustful. I have also noticed that we have services without healthcheck defined. It's a bit like eventual consistency - the policy is already setup, agent is ready, but there are no data. Eventually the service will boot up and metrics data will flow.